cms/core/blocks/tests/factories.py

```diff
@@ -0,0 +1,15 @@
+import factory
```

Comment:
this is extraneous

Reply:
Ahhh, we got rid of this in the main menu PR. Removed.